docs/UsersGuide.asciidoc

```diff
@@ -1334,6 +1334,52 @@ This also works for sub-commands(e.g. `/usr/share/openqa/script/openqa minion -h
 Note that `prefork` is only supported for the main web service but not for
 other services like the live view handler.
 
+== Suggested workflow for test review
+
+If an openQA instance is only used by one or few individuals often no strict
+process needs to defined how openQA tests should be reviewed and how
+individual results should be handled. If the group of test reviewers grows
+openQA and the ecosystem around openQA offer some helpful features and
+approaches.
+
+In particular for a big user base It is important to formalize how decisions
+are made and how tasks are delegated. For this structured comments on the
+openQA platform can be used. With a comment on openQA in the right format one
+can make a decision, inform automatic tools at the same time as other users and
+have a traceable documentation of the actions taken.
+
+* In openQA parent job groups can be defined with multiple job groups. This
+  allows to segment tests for scopes of individual review teams. The parent
+  job group overview pages as well as the central index page of openQA show
+  "bullet list" icons that brings you directly to a combined test overview
+  showing results from all sub groups. This allows to have queries ready like
+  https://openqa.opensuse.org/tests/overview?groupid=1&groupid=2&groupid=3
+  which show all openQA test failures within the hierarchy of test results.
+  This can be combined with the flag "todo=1" (click the "TODO" checkbox in
+  the filter box on test overview pages) to show only tests that need review.
+  Other combinations of queries are possible, e.g.
+  https://openqa.opensuse.org/tests/overview?build=my-build&todo=1 to show
+  all test results that need review for build "my-build"
+* github.com/os-autoinst/openqa_review can be used to produce multiple
```

Comment:
It would be nice to make this a link. The same counts for other occurrences.

Reply:
I added `https://` here which should suffice. Plain URLs are rendered correctly by asciidoctor. We only need the `$url[$label]` syntax when we want to "hide" a complicated URL but in other cases like http://open.qa/docs/#repo-urls we also just use plain URLs. I would use plain URLs if they are still readable. What's your preference here?